examples/storm-starter/pom.xml

```diff
@@ -177,6 +185,36 @@
       <artifactId>storm-redis</artifactId>
       <version>${project.version}</version>
     </dependency>
+      <dependency>
+          <groupId>org.apache.hbase</groupId>
```

Comment:
why is hbase-server dependency needed ? anyways storm-hbase includes hbase-server dependency so this could be removed.